# Clase 15

## Solución de algunos ejercicios de la Tarea 4

<img src="p29-14.png" align="middle" width="600">

<img src="f29-11.png" align="middle" width="300">

- **La forma de menor esfuerzo de hacerlo:**

$$ p=qd=[10(1.6\times10^{-19}\ \text{C})](0.96\times10^{-10}\ \text{m}) $$

In [2]:
p_chafa = 10*1.6*0.96e-29
p_chafa

1.536e-28

$$ 1\ \text{D}=3.33564\times10^{-30}\ \text{C}\cdot\text{m} $$

In [3]:
p_chafa/(3.33564e-30)

46.048134690793965

* **Un cálculo un poco mejor (con la proyección):**

$$ p=q\ell=qd\cos\left(\theta/2\right)=[10(1.6\times10^{-19}\ \text{C})](0.96\times10^{-10}\ \text{m})\cos(52^\circ)$$

In [4]:
p_chafa1 = 10*1.6*0.96e-29*cosd(52)
p_chafa1

9.456560261002111e-29

In [5]:
p_chafa1/(3.33564e-30)

28.350062539728842

* **El mejor cálculo (que se le ocurrió al ayudante):**

https://en.wikipedia.org/wiki/Electric_dipole_moment

$$ \mathbf {E}_1 \left(\mathbf {R} \right)={\frac {3\left(\mathbf {p}_1 \cdot {\hat {\mathbf {R} }}\right){\hat {\mathbf {R} }}-\mathbf {p}_1 }{4\pi \varepsilon _{0}R^{3}}} $$

$$ \mathbf {E}_2 \left(\mathbf {R} \right)={\frac {3\left(\mathbf {p}_2 \cdot {\hat {\mathbf {R} }}\right){\hat {\mathbf {R} }}-\mathbf {p}_2 }{4\pi \varepsilon _{0}R^{3}}} $$

$$ \mathbf {E} \left(\mathbf {R} \right)={\frac {3\left(\mathbf {p}_1 \cdot {\hat {\mathbf {R} }}\right){\hat {\mathbf {R} }}-\mathbf {p}_1 }{4\pi \varepsilon _{0}R^{3}}}+{\frac {3\left(\mathbf {p}_2 \cdot {\hat {\mathbf {R} }}\right){\hat {\mathbf {R} }}-\mathbf {p}_2 }{4\pi \varepsilon _{0}R^{3}}} $$

$$ ={\frac {3\left((\mathbf {p}_1+\mathbf {p}_2) \cdot {\hat {\mathbf {R} }}\right){\hat {\mathbf {R} }}-(\mathbf {p}_1+\mathbf {p}_2) }{4\pi \varepsilon _{0}R^{3}}} $$

$$ \mathbf {p} =\sum _{i=1}^{N}\ q_{i}\mathbf {d_{i}} $$

$$ p=2ed\cos\left(\dfrac{\theta}{2}\right)=2(1.6\times10^{-19}\ \text{C})(0.96\times10^{-10}\ \text{m})\cos(52^\circ) $$

In [6]:
p_chido = 2*1.6*0.96e-29*cosd(52)
p_chido

1.8913120522004224e-29

In [7]:
p_chido/(3.33564e-30)

5.670012507945769

https://en.wikipedia.org/wiki/Properties_of_water

https://en.wikipedia.org/wiki/Properties_of_water#Polarity_and_hydrogen_bonding

https://en.wikipedia.org/wiki/Properties_of_water#cite_ref-FOOTNOTELide20039%E2%80%94Dipole_Moments_18-0

$ p_\text{chido}\approx 18.9\times10^{-30}\ \text{C}\cdot\text{m} = 5.67\ \text{D} $

$ p_\text{real}=6.1\times10^{-30}\ \text{C}\cdot\text{m}=1.83\ \text{D} $

$$ \circledast $$

<img src="p29-34.png" align="middle" width="600">

$$ [k]=\dfrac{\text{carga}}{\text{longitud}^2}\quad,\quad[k_e]=\dfrac{\text{longitud}^2\cdot\text{fuerza}}{\text{carga}^2}=\dfrac{\text{N}\cdot\text{m}^2}{\text{C}^2}\quad\text{(en el SI)} $$

<img src="f29-32.png" align="middle" width="250">

(a)

Las coordenadas del punto $P$ son $(0,y)$.

Tendremos una carga infinitesimal sobre la varilla $ dq $ en las coordenadas $(x,0)$ donde $0\leq x\leq L$.

Entonces el potencial es

$$ dV=k_e\dfrac{dq}{\sqrt{x^2+y^2}} $$

$\lambda$ es la densidad lineal de carga, $\lambda=kx$.

El elemento diferencial de carga se puede expresar como
$$ dq=\lambda\, ds=(kx)(dx) $$

$$ \int_{x=0}^{x=L}k_e\dfrac{kx\,dx}{\sqrt{x^2+y^2}} $$

Sustitución $u=x^2+y^2$,
$$ \dfrac{du}{dx}=2x$$

$$ V(y)=k_ek\dfrac{1}{2}\int_{u(0)}^{u(L)}\dfrac{du}{u^{1/2}}=\left.k_eku^{1/2}\right|_{u(0)}^{u(L)}=k_ek\left(\sqrt{L^2+y^2}-y\right) $$

(b)

¿Cuál es la relación entre el campo y el potencial eléctrico?

$$ \vec{E}=-\nabla V=-\dfrac{\partial V}{\partial x}\hat{x}-\dfrac{\partial V}{\partial y}\hat{y}-\dfrac{\partial V}{\partial z}\hat{z} $$

$$ E_y = -\dfrac{\partial V}{\partial y}=k_ek\left(1-\dfrac{y}{\sqrt{L^2+y^2}}\right) $$

Ahora vamos a hacerlo directo:

Las coordenadas del punto $P$ son $(0,y)$.

Tendremos una carga infinitesimal sobre la varilla $ dq $ en las coordenadas $(x,0)$ donde $0\leq x\leq L$.

Entonces el campo eléctrico es

$$ d\vec{E}=k_e\dfrac{dq}{x^2+y^2}\hat{r}=k_e\dfrac{kx\,dx}{(x^2+y^2)^{3/2}}(-x\hat{x}+y\hat{y}) $$

La componente vertical será

$$ E_y=k_ek\int_{x=0}^{x=L}\dfrac{xy\,dx}{(x^2+y^2)^{3/2}} $$

Sustitución $u=x^2+y^2$,
$$ \dfrac{du}{dx}=2x$$

$$ E_y=k_eky\,\dfrac{1}{2}\int_{u(0)}^{u(L)}\dfrac{du}{u^{3/2}}=\left.k_eky\dfrac{1}{2}\dfrac{u^{-1/2}}{-\frac{1}{2}}\right|_{u(0)}^{u(L)}=-\left.k_eky\dfrac{1}{\sqrt{x^2+y^2}}\right|_{0}^{L} $$

$$ =-k_eky\left(\dfrac{1}{\sqrt{L^2+y^2}}-\dfrac{1}{\sqrt{y^2}}\right)=k_ek\left(1-\dfrac{y}{\sqrt{L^2+y^2}}\right) $$

La componente horizontal será

$$ E_x=k_ek\int_{x=0}^{x=L}\dfrac{x^2\,dx}{(x^2+y^2)^{3/2}} $$

Sustitución $x/y=\tan\theta$,
$$ \dfrac{dx}{d\theta}=y\sec^2\theta$$

$$ E_x=k_ek\int\dfrac{\tan^2\theta\,\sec^2\theta\,d\theta}{(1+\tan^2\theta)^{3/2}}=k_ek\int\sin^2\theta\cos\theta d\theta=k_ek\,\dfrac{1}{3}\left(\dfrac{L}{\sqrt{L^2+y^2}}\right)^3 $$

(c)

¿Cuánto es
$$ \dfrac{d}{dx}\int_a^bf(x)\,dx $$
?

$$ \dfrac{d}{dx}\int_a^bf(x)\,dx=0 $$
porque se trata de una integral definida en límites reales. En cambio,

$$ \dfrac{d}{dx}\int_a^xf(x')\,dx'=f(x) $$

Recordemos que $x'$ es una variable muda, es decir, al hacer la integral desaparece y, por lo tanto, podríamos cambiarla por otra variable:

$$ \int_a^bf(x)\,dx=\int_a^bf(y)\,dy=\int_a^bf(z)\,dz $$

$$ \circledast $$

# Repaso

Flujo eléctrico

$$ \Phi_E=\int \vec{E}\cdot d\vec{S}=\int \vec{E}\cdot\hat{n}\,dS $$

Ley de Gauss: *El flujo a través de una superficie cerrada es proporcional a la carga neta encerrada en dicha superficie.*

$$ \Phi_E=\dfrac{q_\text{neta}}{\varepsilon_0} $$

$$ \Phi_E\equiv\oint \vec{E}\cdot d\vec{S} $$

<img src="tshirt.jpeg" align="middle" width="300">

**Relación entre campo eléctrico y potencial:**

$$ \vec{E}(\vec{r})=-\nabla V(\vec{r}) $$

$$ V(\vec{r})=-\int \vec{E}\cdot d\vec{r} $$

$$ \circledast $$

Sugiero que revisen el sitio preguntando dudas. Hay respuestas muy interesantes:

https://physics.stackexchange.com/